scenarios/kubernetes_bazel.py

```diff
@@ -52,15 +56,58 @@ def get_version():
 
 def main(args):
     """Trigger a bazel build/test run, and upload results."""
+    # pylint:disable=too-many-branches
+    if args.install:
```

Comment:
is it? https://github.com/kubernetes/test-infra/blob/master/jobs/pull-test-infra-bazel.sh#L24-L29 still exists?